[2025-02-14T22:51:51.304+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T22:51:51.331+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T22:51:49.544602+00:00 [queued]>
[2025-02-14T22:51:51.349+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T22:51:49.544602+00:00 [queued]>
[2025-02-14T22:51:51.351+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T22:51:51.394+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_owners> on 2025-02-14 22:51:49.544602+00:00
[2025-02-14T22:51:51.402+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T22:51:51.404+0000] {standard_task_runner.py:72} INFO - Started process 99 to run task
[2025-02-14T22:51:51.408+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'get_owners', 'manual__2025-02-14T22:51:49.544602+00:00', '--job-id', '311', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdzd0qfjf']
[2025-02-14T22:51:51.409+0000] {standard_task_runner.py:105} INFO - Job 311: Subtask get_owners
[2025-02-14T22:51:51.474+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T22:51:49.544602+00:00 [running]> on host 71404c6b5320
[2025-02-14T22:51:51.641+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='get_owners' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T22:51:49.544602+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T22:51:49.544602+00:00'
[2025-02-14T22:51:51.642+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T22:51:51.644+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/***/dags
[2025-02-14T22:51:51.657+0000] {python.py:240} INFO - Done. Returned value was: ['data-team', 'dados']
[2025-02-14T22:51:51.679+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T22:51:51.680+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=get_owners, run_id=manual__2025-02-14T22:51:49.544602+00:00, execution_date=20250214T225149, start_date=20250214T225151, end_date=20250214T225151
[2025-02-14T22:51:51.700+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T22:51:51.722+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T22:51:51.724+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
